src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationStrategy;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private OperationStrategy operationStrategy;
```

Comment:
You test a BalanceOperationHandler, so you don't need a strategy here.
Your tests must be isolated